crates/red_knot_python_semantic/src/types/infer.rs

```diff
@@ -2227,6 +2233,11 @@ impl<'db> TypeInferenceBuilder<'db> {
         let right_ty = self.infer_expression(right);
 
         match (left_ty, right_ty, op) {
+            // When interacting with Todo, Any & Unknown should propagate (as if we fix this Todo
```

Comment:
Best not to use `&` here as it could be misinterpreted as meaning "the intersection type of `Any` and `Unknown` 😄

```suggestion
            // When interacting with Todo, Any and Unknown should propagate (as if we fix this Todo
```